### ISSUE TITLE
Fatal Exception: java.lang.SecurityException: Not allowed to bind to service Intent

### ISSUE THREAD

**Issue by rocklass** [firebase/firebase-android-sdk#5807]
### Describe your environment

  * Android Studio version: Android Studio Iguana | 2023.2.1
  * Firebase Component: 
```
com.google.firebase:firebase-bom:32.7.3
com.google.firebase:firebase-config
com.google.firebase:firebase-messaging
com.google.firebase:firebase-perf
com.google.firebase:firebase-crashlytic
com.google.firebase:firebase-analytics  
```

#### Steps to reproduce:

Crash seen in Crashlytics:
```
Fatal Exception: java.lang.SecurityException: Not allowed to bind to service Intent { act=10874 cmp=xx.xxxxx/com.google.firebase.sessions.SessionLifecycleService (has extras) }
       at android.app.ContextImpl.bindServiceCommon(ContextImpl.java:1741)
       at android.app.ContextImpl.bindService(ContextImpl.java:1651)
       at android.content.ContextWrapper.bindService(ContextWrapper.java:705)
       at com.google.firebase.sessions.SessionLifecycleServiceBinderImpl.bindToService(SessionLifecycleServiceBinder.kt:52)
       at com.google.firebase.sessions.SessionLifecycleClient.bindToService(SessionLifecycleClient.kt:104)
       at com.google.firebase.sessions.FirebaseSessions$1.invokeSuspend(FirebaseSessions.kt:53)
       at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
       at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at com.google.firebase.concurrent.CustomThreadFactory.lambda$newThread$0(CustomThreadFactory.java:47)
       at java.lang.Thread.run(Thread.java:919)
```
![image](https://github.com/firebase/firebase-android-sdk/assets/7993662/f6bdf85f-6c14-4166-9823-f6d58fa6dbf1)
This is the same issue as #5640 but it was closed.

**Comment by lehcar09** [comment]
Hi @rocklass, thank you for reaching out. This issue should have been fixed in PR #5777 that was included in the recent release of [Firebase BOM 32.8.0](https://firebase.google.com/support/release-notes/android#2024-03-21).  

That said, I'll be closing this case now. Let us know if the issue persists and if we need to reopen this issue. Thanks!

**Comment by rocklass** [comment]
Great news! Thanks for the info.
